packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -190,24 +190,36 @@ export const Webchat = forwardRef((props, ref) => {
   const { theme } = webchatState
   const { initialSession, initialDevSettings, onStateChange } = props
   const isOnline = useNetwork()
-  const [botonicState, saveState, deleteState] = useLocalStorage('botonicState')
+  const getThemeProperty = _getThemeProperty(theme)
+
+  const getStorage = () => {
+    const storage = getThemeProperty('storage', props.storage)
+    if (storage === 'none') return null
+    if (storage === 'sessionStorage') return sessionStorage
+    return localStorage
+  }
```

Comment:
Let's let the user provide the storage class directly. So in their project they can do:

```
webchat = {
   ...,
   storage: window.localStorage
}
```

This will allow for more flexibility in the future to provide custom storages.

If they don't want the state to be saved:
```
webchat = {
   ...,
   storage: null,
}
```
it's more natural than a string with value "none"